pkg/mime/types_test.go

```diff
@@ -254,6 +254,16 @@ func TestProtobufMessageTypes(t *testing.T) {
 			messageType: "gnostic.metrics.Vocabulary",
 			mimeType:    "application/octet-stream;type=gnostic.metrics.Vocabulary",
 		},
+		{
+			kind:        "FieldSet",
+			messageType: "google.cloud.apigeeregistry.v1.apihub.FieldSet",
+			mimeType:    "application/octet-stream;type=google.cloud.apigeeregistry.v1.apihub.FieldSet",
+		},
+		{
+			kind:        "FieldSetDefinition",
+			messageType: "google.cloud.apigeeregistry.v1.apihub.FieldSetDefinition",
+			mimeType:    "application/octet-stream;type=google.cloud.apigeeregistry.v1.apihub.FieldSetDefinition",
+		},
```

Comment:
Were these just missing tests? (they are testing proto artifacts, right?)

Reply:
Yes, there were missing tests.

Reply:
Thanks, since these aren't related to the main PR, could you add a brief comment about that in the commit?